core/src/io.rs

```diff
@@ -167,6 +167,7 @@ impl<T: Metadata, S: Middleware<T>> MetaIoHandler<T, S> {
 
 	/// Handle given request asynchronously.
 	pub fn handle_request(&self, request: &str, meta: T) -> FutureResult<S::Future> {
+		info!("handle_request: {:?}", request);
```

Comment:
it will produce output with default logging, is it desirable?

Reply:
I think we should remove it, we already have `trace` couple of lines below.

Reply:
I dropped the commit, so that existing behavior remains unchanged.